www/data_service/src/services/dataUtils/dataUtils.service.coffee

```diff
@@ -0,0 +1,48 @@
+class DataUtils extends Service
+    constructor: ->
+        return new class DataUtils
+            constructor: ->
```

Comment:
You shouldn't create a new instance for a service, because angular does the new-ing for you for services.

This should be enough:
```js
class DataUtils extends Service
    constructor: ->
```